projboard/templates/projboard/searchArticle/search_article.html

```diff
@@ -0,0 +1,79 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+<head>
+    <link rel="stylesheet" type="text/css" href="{% static 'css/search_article.css' %}">
+    
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />    
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Anton&family=Zilla+Slab:wght@400;700&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+<main>
+    <section>
+
+            <form method="POST" action="">
+                {% csrf_token %}
+                
+                <div class="search_button">
+                    <Label for="titleInput"><h4><strong>Enter Article Title:</strong></h4></Label>
```

Comment:
Consider to change h4 to span/p.
You can validate your html code [here](https://validator.w3.org/)

Reply:
i want it h4 right there , i can remove strong tag if thats the problem.

Reply:
fixed it => h4 now is wraping label not the opposite

Reply:
used @Shalevro2 link to validate my html.